src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
 // write code here
+function main() {
+  const doc = document.documentElement;
+  const list = doc.querySelector('ul');
+
+  const children = Array.from(list.children);
+
+  children.sort(sortFunc);
+
+  for (const obj of children) {
+    list.appendChild(obj);
+  }
+}
```

Comment:
The `main` function currently sorts the list by salary using the `sortFunc` function, which is correct. However, according to the task description, you also need to implement a `getEmployees` function that returns an array of employee objects.